resources/lang/de-CH/messages.php

```diff
@@ -0,0 +1,69 @@
+    'section_defaults' => 'Voreinstellungen Bereiche',
+    'section_defaults_description' => 'Überschreibe in einem bestimmten Bereich die Voreinstellungen der Website für Metadaten, soziale Netzwerke, Sitemap und andere Optionen.',
+    'meta' => 'Meta',
+    'image' => 'Bild',
+    'social' => 'Social Media',
+    'sitemap' => 'Sitemap',
+    'search_engines' => 'Suchmaschinen',
+    'generate_report' => 'Bericht erstellen',
+    'delete_report' => 'Bericht löschen',
+    'report_deleted' => 'Bericht gelöscht',
+    'back_to_reports' => 'Zurück zu Berichte',
+    'first_report' => 'Erstelle deinen ersten Bericht',
+    'report_is_being_generated' => 'Dein Bericht wird gerade erstellt.',
+    'generated' => 'Erstellt',
+    'page_details' => 'Seitendetails',
+    'report_no_results_text' => 'Erstelle Berichte, um bei deiner Website nach Verbesserungsmöglichkeiten bei der Suchmaschinenoptimierung zu suchen.',
+    'generate_your_first_report' => 'Erstelle deinen ersten Bericht',
+    'latest_report_score' => 'Punktestand des letzten Berichts',
+    'source_suggest_placeholder' => 'Wähle oder gib einen Feldnamen ein, der den Wert enthält.',
+    'no_collections' => 'Es gibt keine Sammlungen.',
+    'no_taxonomies' => 'Es gibt keine Taxonomien.',
+    'view_reports' => 'Berichte anzeigen',
+    'delete_reports' => 'Berichte löschen',
+    'edit_site_defaults' => 'Voreinstellungen der Website bearbeiten',
+    'edit_section_defaults' => 'Voreinstellungen der Bereiche bearbeiten',
+    'custom' => 'Benutzerdefiniert',
+    'from_field' => 'Von Feld',
+    'inherit' => 'Vererbt',
+    'disable' => 'Deaktiviert',
+    'after' => 'Nach',
+    'before' => 'Bevor',
+    'hourly' => 'Stündlich',
+    'daily' => 'Täglich',
+    'weekly' => 'Wöchentlich',
+    'monthly' => 'Monatlich',
+    'yearly' => 'Jährlich',
+    'never' => 'Nie',
+
+    'rules' => [
+        'site_name' => 'Der Name der Website muss definiert sein.',
+        'unique_title_site' => 'Jede Seite sollte ein eindeutiges Titel-Tag haben.',
+        'unique_title_page' => 'Das Titel-Tag sollte eindeutig sein.',
+        'unique_title_site_failing' => ':count Seiten mit doppeltem Titel.',
+        'unique_title_page_failing' => ':count Seiten mit «:title» als Titel.',
+        'unique_description_site' => 'Jede Seite sollte eine eindeutige Meta-Beschreibung haben.',
+        'unique_description_page' => 'Die Meta-Beschreibung sollte eindeutig sein.',
+        'unique_description_site_failing' => ':count Seiten mit doppelten Meta-Beschreibungen.',
+        'unique_description_page_failing' => ':count Seiten mit «:description» als Meta-Beschreibung.',
+        'no_underscores_in_url_site' => 'Seiten-URLs sollten keine Underscores enthalten.',
+        'no_underscores_in_url_page' => 'Die URL sollte keine Underscores enthalten.',
+        'no_underscores_in_url_site_failing' => ':count Seiten mit Underscores in ihren URLs.',
```

Comment:
This isn't being pluralized because you only supplied the singular version. You need to supply both using a pipe. See https://github.com/statamic/seo-pro/blob/a3b4be1d61860c96319e90d41c9083c304f73426/resources/lang/en/messages.php#L63

Reply:
@helloDanuk ^ It looks like you force pushed that change in? Does that work okay for you now?

Reply:
Gosh, I forgot to push back the other day. I already added them in https://github.com/statamic/seo-pro/pull/155/commits/cf0d00e18559946ecba51e990308a85093e91d5f. Sorry about that. Pluralization didn't worked locally a few weeks ago. But I haven't tested yet. Did you change something? I will give it a try.

Reply:
It should work with that `|` syntax separating the `singular|plural` translations. We're using it on the english side already, and it works great there. I'll merge your PR here once you confirm it works on your end.

Reply:
No, unfortunately it's not working with `de-CH` (Only singular is picked). However, it's working with `de` and `de_CH` with underscore!

- Does this mean I need to change the folder name to `de_CH` with underscore?
- This would also mean changing the translation folder name in statamic/cms?

Reply:
And yes, I have changed the locale name in config/app.php too. Tested all three variants.

Reply:
@helloDanuk We found the issue, which will be fixed in core. More info here: https://github.com/statamic/cms/pull/3405

This PR will require that you rename `de-CH` to `de_CH`, sorry for the trouble!